lib/common/socket/evloop/epoll.c.h

```diff
@@ -192,6 +233,20 @@ static int update_status(struct st_h2o_evloop_epoll_t *loop)
     return 0;
 }
 
+static void evloop_set_bp(struct st_h2o_evloop_epoll_t *loop, uint64_t usecs, uint64_t budget, uint8_t prefer)
```

Comment:
```suggestion
static void set_busypoll(struct st_h2o_evloop_epoll_t *loop, uint64_t usecs, uint64_t budget, uint8_t prefer)
```
"evloop" is not needed for private functions / do not use abbrevs.